src/components/ExperimentalBadge.jsx

```diff
@@ -0,0 +1,27 @@
+/**
+ *  Copyright 2020 Adobe. All rights reserved.
+ *  This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License. You may obtain a copy
+ *  of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under
+ *  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ *  OF ANY KIND, either express or implied. See the License for the specific language
+ *  governing permissions and limitations under the License.
+ */
+
+import { React } from "react"
+import "@spectrum-css/label"
+
+const ExperimentalBadge = ({ inline }) => (
+  <span
+    class="spectrum-Label spectrum-Label--red"
```

Comment:
@chr0n1x sorry can you change `class` to `className`. That is what we are supposed to use in React.